pkg/broker/broker.go

```diff
@@ -1321,7 +1326,11 @@ func (a AnsibleBroker) AddSpec(spec apb.Spec) (*CatalogResponse, error) {
 		return nil, err
 	}
 	apb.AddSecretsFor(&spec)
-	service := SpecToService(&spec)
+	service, err := SpecToService(&spec)
+	if err != nil {
+		a.log.Debugf("spec was not added due to issue with transformation to serivce - %v", err)
```

Comment:
do we intend to log an error as DEBUG? seems odd.

Reply:
I think that warning might be better. This is an APB gave us bad data which we have not treated as an error before I don't think it should be errorf IMO